infrastructure/repository/maintainer-list.tf

```diff
@@ -0,0 +1,9 @@
+// A list of maintainers to be used as an "allow list" for various GitHub Actions.
+// This allows us to make various "exceptions" for maintainers, such as automatically
+// removing the `needs-triage` label from new Issues and Pull Requests
+//
+resource "github_actions_secret" "maintainer_list" {
+  repository      = "github-actions-test"
```

Comment:
maybe this is leftover from some testing 😅 ? should this be `terraform-provider-aws` instead?

Reply:
🤦 yep! indeed it is! Thanks for catching that! Will get it updated now.